x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java

```diff
@@ -460,7 +460,7 @@ public void testKibanaSystemRole() {
         UpdateProfileDataRequest updateProfileDataRequest = randomBoolean()
             ? new UpdateProfileDataRequest(
                 randomAlphaOfLength(10),
-                Map.of("kibana-" + randomAlphaOfLengthBetween(0, 4), mock(Object.class)),
+                Map.of("kibana" + randomAlphaOfLengthBetween(0, 4), mock(Object.class)),
```

Comment:
I think we also should have a test for the literal `"kibana"` (with no suffix) which is the more likely production scenario.

Reply:
The suffix can be of length `0` because it is `randomAlphaOfLengthBetween(0, 4),` which should cover the literal `kibana`.